5.5/root/usr/bin/run-mysqld

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+source /usr/share/container-scripts/mysql/common.sh
+set -eu
+
+[ -f ${MYSQL_SCRIPTS_PATH}/validate-variables.sh ] && . ${MYSQL_SCRIPTS_PATH}/validate-variables.sh
+
+envsubst < ${MYSQL_SCRIPTS_PATH}/my-base.cnf.template > /etc/my.cnf.d/base.cnf
+envsubst < ${MYSQL_SCRIPTS_PATH}/my-paas.cnf.template > /etc/my.cnf.d/paas.cnf
+
+if [ ! -d "$MYSQL_DATADIR/mysql" ]; then
+  initialize_database "$@"
+else
+  start_local_mysql "$@"
+fi
+
+[ -f ${MYSQL_SCRIPTS_PATH}/passwd-change.sh ] && . ${MYSQL_SCRIPTS_PATH}/passwd-change.sh
+[ -f ${MYSQL_SCRIPTS_PATH}/post-init.sh ] && . ${MYSQL_SCRIPTS_PATH}/post-init.sh
+
+# Restart the MySQL server with public IP bindings
+mysqladmin $admin_flags flush-privileges shutdown
+unset_env_vars
+exec ${MYSQL_PREFIX}/libexec/mysqld --defaults-file=$MYSQL_DEFAULTS_FILE "$@" 2>&1
```

Comment:
I don't think so, the container-entrypoint script is already exec'ing this particular script.